CHANGELOG.md

```diff
@@ -12,31 +12,39 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
-- Support for build_info metrics in Prometheus (#35)
-- OpenTelemetry Support (#28)
-- Fly.io example (#26)
-- Django example (#22)
+-
 
 ### Changed
 
 -
 
 ### Deprecated
+
 -
 
 ### Removed
 
-- 
+-
 
 ### Fixed
 
-- The `autometrics` decorator now supports async functions (#33)
-
 ### Security
 
--
+- ***
 
----
+## [0.5](https://github.com/autometrics-dev/autometrics-py/releases/tag/0.5) - 2023-05-11
+
+### Added
+
+- Support `build_info` metrics for Prometheus tracker (#35)
+  - **NOTE**: OpenTelemetry support is not yet supported, see #38
```

Comment:
The language for this note is a bit funky. With the current implementation what is the problem? Should people turn of open telemetry support for the library (i.e. set the AUTOMETRICS_TRACKER env var to "prometheus")?

Reply:
true! i fixed it - it's supposed to convey that the default OpenTelemetry tracker will not work with the build_info metrics